man/virt-install.rst

```diff
@@ -929,6 +929,8 @@ Some examples:
 ``--boot cdrom,fd,hd,network``
     Set the boot device priority as first cdrom, first floppy, first harddisk,
     network PXE boot.
+    Note: s390x guests only support one boot device, so everything except
+    the first device type will be ignored.
```

Comment:
I tried and both `--boot fd,hd` and `--boot network,hd` boot from hd given with the `--disk` argument.

Maybe change to
```
Note: For s390x guests booting from `cdrom` further boot devices will be ignored.
```

Reply:
Well, there are no floppy disks on s390x, so "fd" certainly gets ignored. But with `--boot network,hd` the guest only tries to boot via network for me. What's your full command line? ... maybe you've got something else in there that gets into the way?

Reply:
> Well, there are no floppy disks on s390x, so "fd" certainly gets ignored. But with `--boot network,hd` the guest only tries to boot via network for me. What's your full command line? ... maybe you've got something else in there that gets into the way?

I tried several, e.g.

```
virt-install --boot network,hd --disk size=10 --memory 2048 --vcpus 2 --location <install_tree_url> --os-variant rhel9-unknown
```

```
virt-install --boot network,hd --import --disk <bootable_disk_path> --vcpus 2 --memory 2048 --os-variant rhel8-unknown
```

In both cases there is no tftp server available in the default network, I assumed that's why it falls back to disk.

Reply:
Clarified offline: there was a misunderstanding on my side. The doc update refers to setting after installation.